shopping.html

```diff
@@ -4,10 +4,35 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Shopping</title>
+    <!-- link for the bootsrtap -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <!-- link for the main css file -->
     <link rel="stylesheet" href="./main.css">
 </head>
 <body>
+    <!-- NavBar-->
+
+    <!-- NavBar-->
+    <!-- Main Start here -->
     <h1>Shopping with Pina Cullotte-a</h1>
+    <!-- TODO Div for the Buttons -->
+
+    <!-- TODO Div for the Check out or the Cart button.
+         When click it will show this div and hide the other div -->
+
+         
+    <!-- TODO Div for the Product Details <<Also this div is hide>>
+         it show only if someone click on the product name -->
+
+
+    <!-- TODO Div for the Product Cards this will first show in the page when you landing 
+        << you will hide this only if the user click on Cart for check out -->
+        
+
+    <!-- Main End here -->
+    <!-- Footer-->
```

Comment:
Please add the necessary code for the footer.  That code can be found in the index.html page in the master branch.

Additionally, by the time this PR is being reviewed, there may also be a section in the README with this code.